Shopist/Shopist/Scenes/CheckoutViewController.swift

```diff
@@ -55,38 +62,73 @@ final class CartViewController: UITableViewController {
 
         let dismissButton = UIBarButtonItem(image: UIImage(systemName: "chevron.down"), style: .plain, target: self, action: #selector(dismissPage))
         navigationItem.leftBarButtonItem = dismissButton
-        let payButton = UIBarButtonItem(image: UIImage(systemName: "creditcard"), style: .plain, target: self, action: #selector(pay))
-        navigationItem.rightBarButtonItem = payButton
     }
 
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
         rum?.startView(viewController: self)
-        setupModels(from: cart)
+        if !hasOngoingComputation {
+            hasOngoingComputation = true
+            cart.generateBreakdown {
+                logger.info("Cart is shown with \($0.products.count) items")
+                self.hasOngoingComputation = false
+                self.setupModels(from: $0)
+            }
+        }
+
+        api.fakeUpdateInfoCall()
+        api.fakeFetchFontCall()
+    }
+
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        viewDidAppearDate = Date()
     }
 
     override func viewWillDisappear(_ animated: Bool) {
         super.viewWillDisappear(animated)
         rum?.stopView(viewController: self)
```

Comment:
When Checkout RUM View gets closed, the underlying view is not started, resulting with no active RUM View until some navigation action is taken:

<img width="910" alt="Screenshot 2020-09-02 at 11 12 59" src="https://user-images.githubusercontent.com/2358722/91963215-dd37c980-ed0d-11ea-99e3-21af2b950767.png">